### PR TITLE
Update documentation for iOS notification triggers

### PR DESCRIPTION
**Pull request by aurimasc**
https://jira.unity3d.com/browse/DOCR-4794
Docs for property values and some small updates.

### CODE REVIEW

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 1]
```diff
@@ -79,31 +78,37 @@ public Vector2 Center
...
         /// <summary>
         /// The radius (measured in meters) that defines the geographic area’s outer boundary.
         /// </summary>
+        /// <value>Radius in meters</value>
         public float Radius { get; set; }
 
         /// <summary>
         /// When this property is enabled, a device crossing from outside the region to inside the region triggers the delivery of a notification
         /// </summary>
+        /// <value>True to notify on entering the region</value>
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:99]:
```suggestion
        /// <value>If true, a notification triggers when entering the region.</value>
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 2]
```diff
@@ -79,31 +78,37 @@ public Vector2 Center
...
         public bool NotifyOnEntry { get; set; }
 
         /// <summary>
         /// When this property is enabled, a device crossing from inside the region to outside the region triggers the delivery of a notification
         /// </summary>
+        /// <value>True to notify on leaving the region</value>
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:105]:
```suggestion
        /// <value>If true, a notification triggers when leaving the region.</value>
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 3]
```diff
@@ -79,31 +78,37 @@ public Vector2 Center
...
         public bool NotifyOnExit { get; set; }
 
         /// <summary>
         /// Whether the notification should repeat.
         /// </summary>
+        /// <value>True is notification should repeat</value>
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:111]:
```suggestion
        /// <value>If true, the notification repeats.</value>
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 4]
```diff
@@ -158,6 +160,7 @@ public TimeSpan TimeInterval
         /// <summary>
         /// Whether the notification should repeat.
         /// </summary>
+        /// <value>True if notification should repeat with the same interval</value>
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:163]:
```suggestion
        /// <value>If true, the notification repeats with the same interval.</value>
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 5]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
     {
-        /// <summary>
-        /// The type of notification trigger.
-        /// </summary>
+        /// <inheritdoc/>
         public iOSNotificationTriggerType Type { get { return iOSNotificationTriggerType.Calendar; } }
 
         /// <summary>
-        /// Year
+        /// Year to schedule notification on or null for any year.
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:180]:
```suggestion
        /// Specify the year to schedule the notification for or set to null to indicate any year.
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 6]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
-        /// Second
+        /// Second to schedule notification on or null for any second.
         /// </summary>
+        /// <value>Number indicating second or null to ignore second</value>
         public int? Second { get; set; }
 
         /// <summary>
         /// Are Date and Time field in UTC time. When false, use local time.
         /// </summary>
+        /// <value>True to use UTC time</value>
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:218]:
```suggestion
        /// <value>If true, use UTC time.</value>
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 7]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
         public bool UtcTime { get; set; }
 
         /// <summary>
         /// Indicate whether the notification is repeated every defined time period. For instance if hour and minute fields are set the notification will be triggered every day at the specified hour and minute.
         /// </summary>
+        /// <value>True to repeat notification for each matching date and time, False to only show it on first match</value>
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:224]:
```suggestion
        /// <value>If true, the notification repeats for each matching date and time. If false, the notification triggers only on the first match.</value>
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs (hunk 5 above)
Comment (anjalin14) [reply com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:180]: is this correct?

Comment (aurimasc) [reply com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:180]: Yes.

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 8]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
         /// <summary>
-        /// Year
+        /// Specify the year to schedule the notification for or set to null to indicate any year.
         /// </summary>
+        /// <value>Number indicating year or null to ignore year</value>
         public int? Year { get; set; }
 
         /// <summary>
-        /// Month
+        /// Month to schedule notification on or null for any month.
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:186]:
```suggestion
        /// Specify the month to schedule the notification for or set to null to indicate any month.
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 9]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
         /// </summary>
+        /// <value>Number indicating month or null to ignore month</value>
         public int? Month { get; set; }
 
         /// <summary>
-        /// Day
+        /// Day to schedule notification on or null for any day.
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:192]:
```suggestion
        /// Specify the day to schedule the notification for or set to null to indicate any day.
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 10]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
         /// </summary>
+        /// <value>Number indicating day or null to ignore day</value>
         public int? Day { get; set; }
 
         /// <summary>
-        /// Hour
+        /// Hour to schedule notification on or null for any hour.
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:198]:
```suggestion
        /// Specify the hour to schedule the notification for or set to null to indicate any hour.
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 11]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
         /// </summary>
+        /// <value>Number indicating hour or null to ignore hour</value>
         public int? Hour { get; set; }
 
         /// <summary>
-        /// Minute
+        /// Minute to schedule notification on or null for any minute.
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:204]:
```suggestion
        /// Specify the minute to schedule the notification for or set to null to indicate any minute.
```

File: com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs [hunk 12]
```diff
@@ -170,49 +173,55 @@ public TimeSpan TimeInterval
...
         /// </summary>
+        /// <value>Number indicating minute or null to ignore minute</value>
         public int? Minute { get; set; }
 
         /// <summary>
-        /// Second
+        /// Second to schedule notification on or null for any second.
```
Comment (anjalin14) [inline com.unity.mobile.notifications/Runtime/iOS/iOSNotificationTriggers.cs:210]:
```suggestion
        /// Specify the second to schedule the notification for or set to null to indicate any second.
```

**Review by anjalin14 — APPROVED** [review]
Have left a few more suggestions in line with the earlier suggestion on Year.

**Pull request merged**